gateway-js/src/index.ts

```diff
@@ -742,11 +817,11 @@ export class ApolloGateway implements GraphQLService {
       );
     }
 
-    return getServiceDefinitionsFromStorage({
+    return loadCsdlFromStorage({
       graphId: this.apolloConfig!.graphId!,
-      apiKeyHash: this.apolloConfig!.keyHash!,
+      // TODO: remove TS !
```

Comment:
?

Reply:
I think I hoped to refine the type of `this.apolloConfig` here to assert its truthiness but I'm not sure what I had in mind at the time. Just removing the TODO for now and leaving this as is.